tools/mcsema_disass/ida/get_cfg.py

```diff
@@ -1288,7 +1290,9 @@ def identify_external_symbols():
           del EMAP[target_name]
 
     elif is_external_segment_by_flags(ea) or is_runtime_external_data_reference(ea):
-      extern_name = get_true_external_name(name)
+      # idc.Demangle (...) gives incorrect name for the external data objects
+      # only de-mangle the name of the external functions
+      extern_name = get_true_external_name(name) if is_code(ea) else name
```

Comment:
I think we should instead have a `demangle=True` keword argument to `get_true_external_name`, then pass in `demangle=False` here.